Explain `draw_line` and `draw_rect` coordinates (#6455)
godotengine/godot-docs: tutorials/2d/custom_drawing_in_2d.rst

        }
    }

Coordinates
-----------

> The drawing API uses the CanvasItem coordinate system, not pixel coordinates.
> The CanvasItem's transform is applied. Also, you can use
> :ref:`draw_set_transform<class_CanvasItem_method_draw_set_transform>` or
> :ref:`draw_set_transform_matrix<class_CanvasItem_method_draw_set_transform_matrix>`
> to apply a custom transform additionally.

Reviewer: ```suggestion
The drawing API uses the CanvasItem's coordinate system, not necessarily pixel
coordinates. Which means it uses the coordinate space created after applying
the CanvasItem's transform. Additionally, you can use
:ref:`draw_set_transform<class_CanvasItem_method_draw_set_transform>` or
:ref:`draw_set_transform_matrix<class_CanvasItem_method_draw_set_transform_matrix>`
to apply a custom transform.
```

Author: Done.

> When using ``draw_line``, you should consider the width of the line.
> Lines with odd widths should be shifted by ``0.5``.

Reviewer: ```suggestion
When using ``draw_line``, you should consider the width of the line.
When using a width that is an odd size, the position should be shifted
by ``0.5`` to keep the line centered as shown below.
```

Author: Done.

        }
    }

Coordinates
-----------

> The drawing API uses the CanvasItem's coordinate system, not necessarily pixel
> coordinates. Which means it uses the coordinate space created after applying
> the CanvasItem's transform. Additionally, you can use
> :ref:`draw_set_transform<class_CanvasItem_method_draw_set_transform>` or
> :ref:`draw_set_transform_matrix<class_CanvasItem_method_draw_set_transform_matrix>`
> to apply a custom transform.

Reviewer 2: ```suggestion
The drawing API uses the CanvasItem's coordinate system, not necessarily pixel
coordinates. Which means it uses the coordinate space created after applying
the CanvasItem's transform. Additionally, you can apply a custom transform on
top of it by using
:ref:`draw_set_transform<class_CanvasItem_method_draw_set_transform>` or
:ref:`draw_set_transform_matrix<class_CanvasItem_method_draw_set_transform_matrix>`.
```
(not sure if it should be formatted like I've suggested)

I'd reword it to make it more obvious that the additional custom transform is applied on top of the original transform (instead of e.g. replacing it).

cc @clayjohn to tell if it looks fine for you.

Reviewer: Looks fine to me! I agree that it is more clear
